go/src/koding/klient/machine/index/node/entry.go

```diff
@@ -67,16 +67,16 @@ func (ep *EntryPromise) Swap(set, unset EntryPromise) {
 // system. The fields of this type can be stored on disk and transferred over
 // the network.
 type File struct {
-	CTime int64       `json:"c"` // Metadata change time since EPOCH.
-	MTime int64       `json:"m"` // File data change time since EPOCH.
-	Size  int64       `json:"s"` // Size of the file.
-	Mode  os.FileMode `json:"o"` // File mode and permission bits.
+	CTime int64       `json:"c"`           // Metadata change time since EPOCH.
+	MTime int64       `json:"m"`           // File data change time since EPOCH.
+	Size  int64       `json:"s"`           // Size of the file.
+	Mode  os.FileMode `json:"o"`           // File mode and permission bits.
+	Inode uint64      `json:"i,omitempty"` // Inode ID of a mounted file.
```

Comment:
I see the `wip` tag, but since we started reviewing I just wanted to mention we'd need some compat code for old mounts - https://asciinema.org/a/a0v2txto44hca6wk7ixjpke04.